source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RichTextBlockRenderer.java

```diff
@@ -132,7 +134,10 @@ else if ((textRun = TextRun.dynamic_cast(inline)) == null)
                 TextWeight textWeight = TextRendererUtil.computeTextWeight(hostConfig, TextStyle.Default, textRun.GetTextWeight(), renderArgs);
                 boolean isSubtle = TextRendererUtil.computeIsSubtle(hostConfig, TextStyle.Default, textRun.GetIsSubtle(), renderArgs);
                 FontType fontType = TextRendererUtil.computeFontType(hostConfig, TextStyle.Default, textRun.GetFontType(), renderArgs);
-
+                if (textRun.GetLanguage() == null && textRun.GetLanguage().isEmpty())
```

Comment:
nit: changing to || seems to be better approach here.

Reply:
Oops. Thats should be changed to "||"